src/index.ts

```diff
@@ -107,19 +108,36 @@ export const DEFAULT_OPTIONS: Options = {
   unknownAny: true,
 }
 
+function isYml(filename: string) {
+  return filename.split('.').pop() === 'yml'
```

Comment:
1. Please use `.endsWith`
2. We should support `.yaml` too

```ts
return filename.endsWith('.yaml') || filename.endsWith('.yml')
```

Reply:
Updated as suggested